src/main/kotlin/racing/RacingGame.kt

```diff
@@ -0,0 +1,23 @@
+package racing
+
+import racing.domain.Cars
+import racing.domain.Movable
+import racing.domain.RandomMove
+
+class RacingGame(
+    private val numberOfCars: Int = 2,
+    private var countOfTry: Int = 1,
+    private val movable: Movable = RandomMove()
+) {
+
+    private val cars: Cars = Cars(numberOfCars)
+
+    fun race() = run {
```

Comment:
혹시 해당 함수가 반환하는 값이 무엇인지 알고계신가요? 👀 
제 생각에는 반환 타입이 없는 함수를 의도하셨을 것 같은데, 실제 결과 값은 어떨까요? 😃

저는 해당 함수가 run scope함수를 굳이 사용할 필요가 없다고 생각해요. 🤔
```suggestion
    fun race() {
```

[kotlin 공식 문서 - run](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/run.html)

Reply:
run scope 함수를 잘 모르고 사용했네요 😢 
반환을 의도한 함수가 아니라서 run은 필요가 없을 것 같습니다 😃